lisa/tools/ping.py

```diff
@@ -20,6 +21,18 @@ class Ping(Tool):
     def command(self) -> str:
         return "ping"
 
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        assert isinstance(
```

Comment:
Don't assume it's Ubuntu, it can be any distro. Use if/else instead of assert.

Reply:
in else return the unsupported os exception.

Reply:
Done as requested.